From: Ben Hutchings <ben@decadent.org.uk>
Date: Wed, 16 Jul 2014 20:50:04 +0100
Subject: writeback: Fix ABI change in 3.14.10
Bug-Debian: https://bugs.debian.org/754902
Forwarded: not-needed

Commit 1c8349a17137 ("ext4: fix data integrity sync in ordered mode")
renamed test_set_page_writeback() to __test_set_page_writeback() and
added a function-macro under the old name as a wrapper.
test_set_page_writeback() was an exported function that is used by
some OOT modules including openafs.  To keep old modules working, we
need to provide a real function as well.

--- a/mm/page-writeback.c
+++ b/mm/page-writeback.c
@@ -2439,6 +2439,13 @@ int __test_set_page_writeback(struct pag
 }
 EXPORT_SYMBOL(__test_set_page_writeback);
 
+#undef test_set_page_writeback
+int test_set_page_writeback(struct page *page)
+{
+	return __test_set_page_writeback(page, false);
+}
+EXPORT_SYMBOL(test_set_page_writeback);
+
 /*
  * Return true if any of the pages in the mapping are marked with the
  * passed tag.
